Easy/DuplicateInterger/solution.js

```diff
@@ -0,0 +1,16 @@
+class Solution {
+    /**
+     * @param {number[]} nums
+     * @return {boolean}
+     */
+    hasDuplicate(nums) {
+        let uniques = new Set();
+        for (let i = 0; i< nums.length ; i++){
+            if(uniques.has(nums[i])){
+                return true;
+            }
+            uniques.add(nums[i])
+        }
+        return false;
```

Comment:
Recomendo instalar no seu vscode o eslint e prettier para conseguir tem um padrão legal de código, seguindo as boas praticas de escrita. Mas lembrando que você tem que bater o olho e já pegar o que pode ser melhora, durante e depois da escrita do código. Minha sugestão seria:

```suggestion
        for (let i = 0; i < nums.length ;  ++i) {
            if (uniques.has(nums[i])) {
                return true;
            }
            
            uniques.add(nums[i]);
        }
        
        return false;
```

Reply:
Vou aprovar para seguir em frente, mas ainda falta o `;`. Sem falar que falta a organização dos if e for com espaços e chaves